Source/WebCore/platform/network/curl/NetworkStorageSessionCurl.cpp

```diff
@@ -232,6 +232,16 @@ void NetworkStorageSession::setProxySettings(const CurlProxySettings& proxySetti
     CurlContext::singleton().setProxySettings(proxySettings);
 }
 
+void NetworkStorageSession::setAlternativeServicesStorageFile(const String& cacheFile) const
+{
+    CurlContext::singleton().setAlternativeServicesStorageFile(cacheFile);
```

Comment:
I think you should be working towards making CurlContext no longer a singleton.  That's what NetworkStorageSession is for.

Reply:
I will fix it in a separate PR.